### ISSUE TITLE
Via fails for text files

### ISSUE THREAD

**Issue by acka47** [hypothesis/via#79]
Unfortunately via doesn't work with text files while the chrome plugin does. Already in December 2015 I encountered this problem, see my [tweet](https://twitter.com/acka47/status/674868213738561536) with response from @robertknight .

Currently, I would like to use hypothes.is for documenting the data provided by our API. I started annotating an [example file](http://lobid.org/organisations/DE-38M?format=json) and would like to link to the annotated example from a documentation page. As I can't add a line of javascript to a JSON file I depend on via for showing the annotations. For now, I will find another solution. It would be great if you added this feature, though, for future use cases.

I also added an [example annotation](https://hyp.is/Scv0VHNBEeatuXtM8KWwiA/raw.githubusercontent.com/hypothesis/via/master/requirements.txt) to requirements.txt in this repo which shows the same behaviour.

**Comment by ntnsndr** [comment]
Agreed that this is an issue. My [course](https://nathanschneider.info/wiki/school/hacker_culture) is assigned to annotate [this text file](http://www.textfiles.com/hacking/eslf0006.txt), for instance. It works with the Firefox bookmarklet but fails on Via.
